src/app/pages/permissions-page/permissions.page.ts

```diff
@@ -141,48 +174,34 @@ export class PermissionsPage {
   }
 
   exitToHomePage() {
-      if (this._permissionsModelService.isDirty() == false){
-        this._permissionsModelService.selectedUserName = "";
-        this._permissionsModelService.selectedUserRoles = [];
-        this._permissionsModelService.hasSelectedUser = false;
-        this.router.navigate(['home']);
-      }
-      else {
-        
-        this._alertService.show({
-          header: 'Changes Not Saved!',
-          message: 'Discard changes?',
-          buttons: [{
-            text: "Go Back",
-            role: 'cancel'
-          }, {
-            text: "Discard" ,
-            handler: () => {
-              this._permissionsModelService.dirty = false;
-              this._permissionsModelService.selectedUserName = "";
-              this._permissionsModelService.selectedUserRoles = [];
-              this._permissionsModelService.hasSelectedUser = false;
-              this.router.navigate(['home']);
-            }
-          }]
-        })
-        
-      }
+    if (this._permissionsModelService.isDirty() == false){
+      this._permissionsModelService.selectedUserName = "";
+      this._permissionsModelService.selectedUserRoles = [];
+      this._permissionsModelService.hasSelectedUser = false;
```

Comment:
Instead, use an `ngOnInit()` method on this page, so when the page loads you can initialize the permissionsModelService.